### DISCUSSION TITLE
v3.2.1

### DISCUSSION THREAD

**Discussion by ccseer** [ccseer/Seer#403] [Relnotes]
- folder: a little bit faster, when previewing C:/Windows/System32, it takes 250ms less than previous version
- video: new plugin for r3d. it needs to be added in a different way
        https://github.com/ccseer/Seer/discussions/399
- video: fixed qr code error
- PDF: new shortcuts for next/prev page,  `]` / `[`
- PDF: fixed shortcuts for Dark/Annotate not working in separated
- user.ini: added "load_cloud_files" under "General"
- user.ini: added "enable_esc_key" under "General", close window globally
- new variables - ${use_backslash}: replace forward slashes with backslashes
- [Files App](https://github.com/files-community) might support Seer in its new version

**Comment by gautambishnoi** [comment]
Great job, Corey!!! 👍
